lib/ansible/modules/system/vdo.py

```diff
@@ -606,14 +606,18 @@ def run_module():
         modtrans = {}
 
         for statfield in statusparamkeys:
-            currentvdoparams[statfield] = processedvdos[desiredvdo][statfield]
-            modtrans[statfield] = vdokeytrans[statfield]
+            if statfield in processedvdos[desiredvdo]:
+                currentvdoparams[statfield] = processedvdos[desiredvdo][statfield]
+                modtrans[statfield] = vdokeytrans[statfield]
+
+            else:
```

Comment:
You can remove the else here and always set ```modtrans[statfield] = vdokeytrans[statfield]``` outside the if block.

Reply:
Ah, okay; I see it now.  Updated.